lldb/include/lldb/lldb-enumerations.h

```diff
@@ -257,8 +257,8 @@ enum StopReason {
 };
 
 /// Command Return Status Types.
-enum ReturnStatus {
-  eReturnStatusInvalid,
+enum ReturnStatus : int {
+  eReturnStatusInvalid = 0,
```

Comment:
carry-over comment from previous patch("Why is this needed?")
 This is needed to serialize the ReturnStatus object